litmus-portal/frontend/src/components/WelcomeModal/styles.ts

```diff
@@ -3,13 +3,7 @@ import { makeStyles, Theme } from '@material-ui/core/styles';
 const useStyles = makeStyles((theme: Theme) => ({
   /* CSS for Index */
   rootContainer: {
-    margin: '3rem auto',
-    width: '60%',
-    height: '85%',
-    background: theme.palette.common.white,
-    borderRadius: 3,
-    textAlign: 'center',
-    outline: 'none',
+    paddingBottom: '8.5rem',
   },
```

Comment:
Would be a lot better if we can use `theme.spacing()` for padding and margins.

Reply:
Okay. I'll change this to `theme.spacing()`. There's a mix of `theme.spacing()` and `rem` used in the styling of separate components which will probably be made consistent later.